### PR TITLE
Fixed [Environment] OpenCV NULL guiReceiver error during pre-commit

### PR DESCRIPTION
**Pull request by Princekumarofficial**
#230 
Fixed [Environment] OpenCV NULL guiReceiver error during pre-commit testing .

Added a check to see if a window is available, or create one if its not.

### CODE REVIEW

File: src/utils/interaction.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ def show(name, origin, pause=1, resize=False, reset_pos=None, config=None):
         else:
             img = origin
 
+        if not is_window_available(name):
+            cv2.namedWindow(name)
```
Comment (Udayraj123) [inline src/utils/interaction.py:42]: can you add this for the cv2.moveWindow() as well?

**Comment by Udayraj123** [comment]
Thanks, added a comment @Princekumarofficial

File: src/utils/interaction.py (hunk 1 above)
Comment (Princekumarofficial) [reply src/utils/interaction.py:42]: Its not needed separately for the same, We can do that but it does not affect anything as we are already handling the case when we don't have window here, so no need to do it again.

**Review by Udayraj123 — APPROVED** [review]
LGTM!

**Pull request merged**